### DISCUSSION TITLE
The `./dist` folder mentioned in CONTRIBUTING.md

### DISCUSSION THREAD

**Discussion by jeroengerits** [tailwindlabs/tailwindcss#6179] [General]
I've recently got interested in HTML class names with arbitrary values, and now - since Tailwind CSS supports them I want to test a few things and see how it all works. The `CONTRIBUTING.md` documentation states the following: 

![image](https://user-images.githubusercontent.com/808734/142922006-879020ba-41cf-4d11-83d0-3f6ebfc3e1e6.png)

However, there is no such `dist` folder after running the above command(s), there is the `lib` folder and the actual root folder which seems to be the actual folder mentioned, but unsure.

As said, I'm not sure about it, and maybe I've missed something - since this is the first time I'm touching Tailwind CSS source code? :thinking: 

If the `root` folder happens to be the correct folder, I'm happy to issue a pull request for the `CONTRIBUTING.md` file and references accordingly.  

In any other case, can someone explain to me how to test the generated CSS with a custom HTML file the 'actual' way you describe? What is the command to generate a `dist` folder? 

**Thanks for all the work!** :love_you_gesture:

**Comment by RobinMalfait** [comment]
Hey! Thank you for your question!
Much appreciated! 🙏

We used to have a `./dist` folder when we only had the `AOT` mode (this is the mode where we generated every css utility upfront and used purge css to remove the ones you didn't use). This dist folder contained generated css files that we also used for our CDN builds.

However, we have introduce a new `JIT` mode, which doesn't generate classes upfront, but uses your content paths to know which utilities you've used. This means that we can't generate anything anymore for our CDN build.

The current `master` branch, is the branch that contains all of Tailwind CSS `v3` (unreleased) and the `CONTRIBUTING.md` file is out of date because of this.

We do have a new solution for the CDN build with `JIT` mode, more information can be found here: https://github.com/tailwindlabs/tailwindcss/releases/tag/v3.0.0-alpha.1#just-in-time-cdn
Hope this helps!

TL;DR: The `CONTRIBUTING.md` file is outdated.

**Reply by jeroengerits** [reply]
Hey Robin, thanks for clarifying. 

Happy to issue a small PR to the `CONTRIBUTING.md` if desired?

Proposal: https://github.com/jeroengerits/tailwindcss/commit/65887b628693252c28684dec8a6afc24b2ae4085
